<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWidget</class>
 <widget class="QWidget" name="AdminWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员面板</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>661</height>
    </rect>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>学生管理</string>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>考勤统计</string>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>设备管理</string>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>1081</width>
     <height>661</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="userManagePage">
    <widget class="QTableWidget" name="infoTableWidget">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>621</width>
       <height>601</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>年级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>院系</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>专业</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>班级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>邮箱</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联系方式</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>新增</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>490</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>编辑</string>
     </property>
    </widget>
    <widget class="QComboBox" name="labCmb">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>530</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>570</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QLabel" name="pictureView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>384</width>
       <height>288</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="logOutButton">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>10</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注销</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="attendancePage">
    <widget class="QTableWidget" name="historyRankTableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>1031</width>
       <height>251</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>128</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>排名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>签到/未签/缺勤率</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>院系</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>年级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>班级</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="historyComboBox">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>350</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>350</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="beginDateEdit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>350</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="endDateLabel">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>354</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>至</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="endDateEdit">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>350</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="todayRankTableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>1031</width>
       <height>281</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>114</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>排名</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>签到时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>院系</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>年级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>专业</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>班级</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="todayLabComboBox">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>10</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="todayLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>今日签到情况：</string>
     </property>
    </widget>
    <widget class="QLabel" name="historyLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>历史签到情况：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="devicesManagePage">
    <widget class="QTableWidget" name="deviceInfoTableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>671</width>
       <height>551</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>133</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>设备名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>借出数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>损坏数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>剩余数量</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="deviceLabComboBox">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="boundCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>620</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>认证该主机到此实验室</string>
     </property>
    </widget>
    <widget class="QListWidget" name="deviceLogListWidget">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>60</y>
       <width>341</width>
       <height>551</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <item>
      <property name="text">
       <string>14::03:56:                电脑借出4台</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="deviceLogDateEdit">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>20</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="deviceLogLabel">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>19</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>日志：</string>
     </property>
    </widget>
    <widget class="QLabel" name="deviceInfoLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>19</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设备信息：</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
